### DISCUSSION TITLE
How to specify port and subnet?

### DISCUSSION THREAD

**Discussion by AndrewEastwood** [ddosify/ddosify#56] [Q&A] [answered]
Im trying to test specific port among few identical docker instances; The subnet is 172.17. 0.0/16 and port is 54
Question: how to start load testing these instances?
Thanks!

**Comment by fatihbaltaci** [comment, marked as answer]
Hi @AndrewEastwood, you can use the following command by changing the IP address:

```bash
ddosify -t http://172.17.0.1:54
```
